### PR TITLE
LRSD-4690 Liferay Self-Hosted subscription cards don't show instance …

### PR DESCRIPTION
**Pull request by amosfong**
…size

### CODE REVIEW

**Comment by amosfong** [comment]
@ryanschuhler can you review, thanks!

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/Overview/components/SubscriptionsOverview/components/AccountSubscriptionsList/components/AccountSubscriptionModal/utils/getColumns.jsx [hunk 1]
```diff
@@ -107,23 +107,23 @@ export default function getColumns(
...
+	].some((category) => {
+	    if (title.startsWith(category)) {
+	        const productName = title.substring(category.length + 1);
 
-	if (displayInstanceSizeByCategories) {
-		const [category, ...product] = title?.split(' ');
-		const productName = product.join(' ');
+	        return displayInstanceSizeMap[
+            	category
+            ].includes(productName);
```
Comment (kylembischof) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/Overview/components/SubscriptionsOverview/components/AccountSubscriptionsList/components/AccountSubscriptionModal/utils/getColumns.jsx:120]: lines after return should be indented

Comment (amosfong) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/Overview/components/SubscriptionsOverview/components/AccountSubscriptionsList/components/AccountSubscriptionModal/utils/getColumns.jsx:120]: oh hm...formatsource doesn't seem to be working. I'll look into this tomorrow

**Comment by amosfong** [comment]
Re-pushed, can you review now, thanks!

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/Overview/components/SubscriptionsOverview/components/AccountSubscriptionsList/components/AccountSubscriptionModal/utils/getColumns.jsx (hunk 1 above)
Comment (amosfong) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/Overview/components/SubscriptionsOverview/components/AccountSubscriptionsList/components/AccountSubscriptionModal/utils/getColumns.jsx]: Going to create another ticket for this. formatsource is not checking our dir

**Review by ryanschuhler — APPROVED** [review]
lgtm

**Pull request closed without merging**